.gitignore

```diff
@@ -0,0 +1,157 @@
+# IDE files / folders
+.idea
+.vscode
```

Comment:
Added lines so that GitHub doesn't ask you to add IDE files to your repository all the time.